images/deno-dev/post-plugin.vim

```diff
@@ -0,0 +1,19 @@
+" Simple debugger integration.
+
+fu! g:SendDBBreak()
```

Comment:
Is there even a cli-based debugger for deno?

Reply:
yes, but I might have it set up wrong, thoughts?

https://deno.land/manual/tools/debugger